eidpdata/src/main/java/uk/ac/ucl/eidp/data/StrategyResolver.java

```diff
@@ -30,70 +15,83 @@
 
 /**
  *
- * @author David Guzman <d.guzman at ucl.ac.uk>
+ * @author David Guzman {@literal d.guzman at ucl.ac.uk}
  */
 @Stateless
 public class StrategyResolver {
 
-    private final Map<String, Properties> databaseNodes = new HashMap<>();
-    private final String NODETYPE_PROP = "uk.ac.ucl.eidp.data.NodeType";
-    private final String CUSTOM_STRATEGY_JNDI = "uk.ac.ucl.eidp.data.DBMappingStrategy";
+  private final Map<String, Properties> databaseNodes = new HashMap<>();
+  private final String nodetypeProperty = "uk.ac.ucl.eidp.data.NodeType";
+  private final String customStrategyJndi = "uk.ac.ucl.eidp.data.DBMappingStrategy";
```

Comment:
\[FindBugs\] INFO: SS: Unread field: uk\.ac\.ucl\.eidp\.data\.StrategyResolver\.customStrategyJndi; should this field be static?